api/src/main/java/org/pmiops/workbench/reporting/insertion/InsertAllRequestPayloadTransformer.java

```diff
@@ -27,11 +28,19 @@
  * @code{ InsertAllRequestPayloadTransformer<ReportingUser> userTransformer =
  *     UserColumnValueExtractor::values; }
  */
+@FunctionalInterface
 public interface InsertAllRequestPayloadTransformer<MODEL_T>
     extends BigQueryInsertionPayloadTransformer<MODEL_T> {
   String INSERT_ID_CHARS = "abcdefghijklmnopqrstuvwxyz";
   int INSERT_ID_LENGTH = 16;
+  int MAX_ROWS_PER_INSERT_ALL_REQUEST = 1000;
```

Comment:
Should this use the `maxRowsPerInsert` config value?

Reply:
It does, at least in the new code. The constant is a cap on the configured value.